src/libs/Network/NetworkStore.ts

```diff
@@ -82,85 +78,51 @@ Onyx.connect({
             triggerReconnectCallback();
         }
 
-        offline = Boolean(network.shouldForceOffline) || network.isOffline;
+        offline = Boolean(network.shouldForceOffline) || !!network.isOffline;
     },
 });
 
-/**
- * @returns {Object}
- */
-function getCredentials() {
+function getCredentials(): Credentials | null {
     return credentials;
 }
 
-/**
- * @returns {Boolean}
- */
-function isOffline() {
+function isOffline(): boolean {
     return offline;
 }
 
-/**
- * @returns {String}
- */
-function getAuthToken() {
+function getAuthToken(): string | null {
     return authToken;
 }
 
-/**
- * @param {String} command
- * @returns {[String]}
- */
-function isSupportRequest(command) {
-    return _.contains(['OpenApp', 'ReconnectApp', 'OpenReport'], command);
+function isSupportRequest(command: string): boolean {
+    return ['OpenApp', 'ReconnectApp', 'OpenReport'].includes(command);
 }
 
-/**
- * @returns {String}
- */
-function getSupportAuthToken() {
+function getSupportAuthToken(): string | null {
     return supportAuthToken;
 }
 
-/**
- * @param {String} newSupportAuthToken
- */
-function setSupportAuthToken(newSupportAuthToken) {
+function setSupportAuthToken(newSupportAuthToken: string) {
     supportAuthToken = newSupportAuthToken;
 }
 
-/**
- * @param {String} newAuthToken
- */
-function setAuthToken(newAuthToken) {
+function setAuthToken(newAuthToken: string | null) {
```

Comment:
Almost sure this needs to be just string, no null

Reply:
`authToken` is optional in `Response` type:
```ts
NetworkStore.setAuthToken(response.authToken ?? null);
```

That's why I allowed null values for this function, but usually it should be just a string, wdyt?